build.gradle

```diff
@@ -2249,7 +2264,7 @@ project(":graphics") {
     dependencies {
         antlr group: "org.antlr", name: "antlr4", version: "4.7.2", classifier: "complete"
         testImplementation project(":base").sourceSets.test.output
-        implementation project(':base')
+        api project(':base')
```

Comment:
Same question here. This seems unrelated to the changes you are trying to make.

Reply:
This is correcting the dependency scopes to be like they are in the `pom` after publishing. Because the information is correct here now, we no longer need the lists of dependency modules in these calls: `addMavenPublication(project, [ 'base' ])` (in this example, the `[base]` is removed here).